Broken build with system liburiparser

### Description

Build is broken using `--with-external-uriparser`

```
/usr/bin/ld: ext/uri/uri_parser_rfc3986.o: in function `php_uri_parser_rfc3986_validate_host':
/builddir/build/BUILD/php86-php-8.6.0_alpha1-build/php-8.6.0alpha1/ext/uri/uri_parser_rfc3986.c:746:(.text+0x148d): undefined reference to `uriIsWellFormedHostIp6A'
/usr/bin/ld: /builddir/build/BUILD/php86-php-8.6.0_alpha1-build/php-8.6.0alpha1/ext/uri/uri_parser_rfc3986.c:740:(.text+0x14dc): undefined reference to `uriIsWellFormedHostIpFutureA'
/usr/bin/ld: ext/uri/uri_parser_rfc3986.o: in function `php_uri_parser_rfc3986_validate_host':
/builddir/build/BUILD/php86-php-8.6.0_alpha1-build/php-8.6.0alpha1/ext/uri/uri_parser_rfc3986.c:746:(.text+0x148d): undefined reference to `uriIsWellFormedHostIp6A'
/usr/bin/ld: /builddir/build/BUILD/php86-php-8.6.0_alpha1-build/php-8.6.0alpha1/ext/uri/uri_parser_rfc3986.c:740:(.text+0x14dc): undefined reference to `uriIsWellFormedHostIpFutureA'
/usr/bin/ld: ext/uri/uri_parser_rfc3986.o: in function `php_uri_parser_rfc3986_validate_host':
/builddir/build/BUILD/php86-php-8.6.0_alpha1-build/php-8.6.0alpha1/ext/uri/uri_parser_rfc3986.c:746:(.text+0x148d): undefined reference to `uriIsWellFormedHostIp6A'
/usr/bin/ld: /builddir/build/BUILD/php86-php-8.6.0_alpha1-build/php-8.6.0alpha1/ext/uri/uri_parser_rfc3986.c:740:(.text+0x14dc): undefined reference to `uriIsWellFormedHostIpFutureA'

```



### PHP Version

```plain
8.6.0alpha1
```

### Operating System

N/A

Thanks for upstream patch, but sadly, still not enough

```
/usr/bin/ld: ext/uri/uri_parser_rfc3986.o: in function `php_uri_parser_rfc3986_validate_host':
/builddir/build/BUILD/php86-php-8.6.0_alpha1-build/php-8.6.0alpha1/ext/uri/uri_parser_rfc3986.c:740:(.text+0x14dc): undefined reference to `uriIsWellFormedHostIpFutureA'
/usr/bin/ld: ext/uri/uri_parser_rfc3986.o: in function `php_uri_parser_rfc3986_validate_host':
/builddir/build/BUILD/php86-php-8.6.0_alpha1-build/php-8.6.0alpha1/ext/uri/uri_parser_rfc3986.c:740:(.text+0x14dc): undefined reference to `uriIsWellFormedHostIpFutureA'
/usr/bin/ld: ext/uri/uri_parser_rfc3986.o: in function `php_uri_parser_rfc3986_validate_host':
/builddir/build/BUILD/php86-php-8.6.0_alpha1-build/php-8.6.0alpha1/ext/uri/uri_parser_rfc3986.c:740:(.text+0x14dc): undefined reference to `uriIsWellFormedHostIpFutureA'

```

--- a/include/uriparser/Uri.h
+++ b/include/uriparser/Uri.h
@@ -1224,9 +1224,9 @@ URI_PUBLIC int URI_FUNC(IsWellFormedHostIp6)(
  * @see uriIsWellFormedQueryA
  * @see uriIsWellFormedSchemeA
  * @see uriIsWellFormedUserInfoA
- * @since 0.9.9
+ * @since 1.0.3
  */
-int URI_FUNC(IsWellFormedHostIp6Mm)(
+URI_PUBLIC int URI_FUNC(IsWellFormedHostIp6Mm)(
         const URI_CHAR * first, const URI_CHAR * afterLast, UriMemoryManager * memory);
 
 /**
@@ -1252,9 +1252,9 @@ int URI_FUNC(IsWellFormedHostIp6Mm)(
  * @see uriIsWellFormedUserInfoA
  * @see uriSetHostIpFutureA
  * @see uriSetHostIpFutureMmA
- * @since 0.9.9
+ * @since 1.0.3
  */
-int URI_FUNC(IsWellFormedHostIpFuture)(
+URI_PUBLIC int URI_FUNC(IsWellFormedHostIpFuture)(
         const URI_CHAR * first, const URI_CHAR * afterLast);
 
 /**
@@ -1279,9 +1279,9 @@ int URI_FUNC(IsWellFormedHostIpFuture)(
  * @see uriIsWellFormedUserInfoA
  * @see uriSetHostIpFutureA
  * @see uriSetHostIpFutureMmA
- * @since 0.9.9
+ * @since 1.0.3
  */
-int URI_FUNC(IsWellFormedHostIpFutureMm)(
+URI_PUBLIC int URI_FUNC(IsWellFormedHostIpFutureMm)(
         const URI_CHAR * first, const URI_CHAR * afterLast, UriMemoryManager * memory);
 
 /**

